charts/harbor-app-operator/templates/rbac.yaml

```diff
@@ -0,0 +1,42 @@
+{{- if .Values.serviceAccount.create }}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ include "harbor-app-operator.serviceAccountName" . }}
+  namespace: {{ .Release.Namespace | quote }}
+automountServiceAccountToken: true
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ include "harbor-app-operator.fullname" . }}-edit-harbor
+rules:
```

Comment:
Could you please write comments that justify RBAC

Reply:
Added!